### ISSUE TITLE
Show page previews when scrolling through full screen reader

### ISSUE THREAD

**Issue by jeroenhd** [jeroenhd/UnofficialBCBReader#22]
When scrolling through the full screen reader, page previews should be visible. The iOS app does this too and the design pattern should work perfectly in material design as well.

The required LqThumbs function in API.java already exists (should be updated, but that's not too hard).

The page display should also feature the chapter title (and maybe the description) of the image centered, and a strip of the next and previous pages should be visible. Something like this should suffice:
![Quick design draw up](https://cloud.githubusercontent.com/assets/12149431/18034858/159b5e9e-6d48-11e6-8988-bd89238430ba.png)

**Comment by jeroenhd** [comment]
Suitcase mentioned a while back (during the stream) that the thumbs are possibly not cached by the CDN and using them could DDOS the BCB server.
Until there's more known about this issue, this idea is better not implemented

**Comment by jeroenhd** [comment]
Forgot to update this issue, but the DDOS problem does not exist.
It would require some dark HTTP(2?) incantations if we don't the scrolling to lag though!

**Comment by jeroenhd** [comment]
After some preliminary tests I conclude that implementing this would cause too much lag on low-end devices. Since low-end devices don't indicate they're low-end, this issue should be closed until the app is finished.
